src/main/resources/alfresco/subsystems/Messaging/default/messaging-context.xml

```diff
@@ -68,6 +68,9 @@
     <!-- Any RouteBuilder in this package will be fired up -->
     <context:component-scan base-package="org.alfresco.messaging.camel.routes"/>
 
+    <!-- Any RouteBuilder in this enterprise package will be fired up -->
+    <context:component-scan base-package="org.alfresco.enterprise.repo.rendition2.routes"/>
```

Comment:
We should change to "org.alfresco.enterprise.repo.routes" so we can reuse it also for other parts and not just renditions.